src/Symfony/Component/Console/Question/Question.php

```diff
@@ -24,8 +24,17 @@ class Question
     private ?int $attempts = null;
     private bool $hidden = false;
     private bool $hiddenFallback = true;
+    /**
+     * @var (\Closure(mixed):iterable)|null
+     */
     private ?\Closure $autocompleterCallback = null;
+    /**
+     * @var (\Closure(mixed):bool)|null
```

Comment:
the validator does not actually return `bool`. It returns the validated value (potentially transformed) or throws an exception.